### ISSUE TITLE
JSON to .d.ts support

### ISSUE THREAD

**Issue by basarat** [alm-tools/alm#112]
@xperiments has an excellent project https://github.com/xperiments/json2dts that converts stuff like: 

```js
{
  foo: 123,
  bar: {
    bas: "Hello",
    baz: {
      qux: "World"
    }
  }
}
```
into TypeScript interfaces like this:

```ts
interface Bar
{
	bas:string;
	baz:Baz;
}

interface Baz
{
	qux:string;
}

interface RootJson
{
	foo:number;
	bar:Bar;
}
```

It would be a shame not to share this with the world, so I'll be integrating it into alm :rose:

**Comment by Alexisvt** [comment]
Nice!! Yes yes do it! 😂😁

**Comment by basarat** [comment]
Released with `0.0.76` :rose:

![jsontodts](https://cloud.githubusercontent.com/assets/874898/15596161/6dcbde92-2408-11e6-95ac-b2307d0a26ed.gif)
